nemo/core/config/optimizers.py

```diff
@@ -15,13 +15,50 @@
 # =============================================================================
 
 from dataclasses import dataclass
-from typing import Tuple
+from functools import partial
+from typing import Any, Dict, Optional, Tuple
+
+__all__ = [
```

Comment:
All can probably be removed since we disallow * imports

Reply:
I thought I had removed all of them...